packages/flutter/test/widgets/dismissible_test.dart

```diff
@@ -12,6 +12,7 @@ const DismissDirection defaultDismissDirection = DismissDirection.horizontal;
 const double crossAxisEndOffset = 0.5;
 bool reportedDismissUpdateReached = false;
 bool reportedDismissUpdatePreviousReached = false;
+double reportedDismissUpdateProgress = 0.0;
```

Comment:
Can you add tests for non-zero cases?

Reply:
Can you add another test that validates it reports the right value somewhere in between 0 and 1? Thanks!

Reply:
Done.